code/modules/food_and_drink/drinks.dm

```diff
@@ -329,13 +329,15 @@
 	heal_amt = 1
 	initial_volume = 50
 	initial_reagents = "milk"
+	var/canbequilty = 1
 
 	New()
 		..()
-		if( prob(10) )
-			name = "Quilty Farms Milk"
-			desc = "For ages 1[pick("0","8")] and under."
-			icon_state = "milk_quilty"
+		if(canbequilty < 1)
```

Comment:
shouldnt this be ==?

Reply:
I don't know. Never seen any errors last time I checked it out.

Reply:
Unless someone complains I would just argue to please just go with it as it is...I wrecked my nerves for one week fixing 2 files and saving the broken VSC and github I have. Please let my soul rest. q_q'

Reply:
< means less then, so this would make milk naturally not be quilty. (and also a bit less important but you can merge this if statement with the one below using a && between them)

Reply:
So...what are the implications of the way I handled it? I have a hard time following you mate.
Like...what is wrong about the way I impemented it?
The problem I had with introducing new types of milk was that even if you spawn them in...like clownspider_milk and cluwne_spider milk that it would sometimes spawn in a quilty bottle instead. That change I added made it possible for all bottles to spawn without over-writing custom ones.

Reply:
?, Why are you being so defensive about this?, im literally trying to help you here. Your implementation would cause normal milk bottles to never be quilty due to it checking for the quilty variable to be less then 1. side effect of this would mean your new milk would have the chance of being quilty

Reply:
Not trying to be defensive or come across like it. Sorry.
Perhaps I phrased myself strangely. :o

Ah yeah,...
One second. Let me check something. I really did not mean to come across the wrong way sorry. D:

Reply:
Wait. Check  line 332.

This causes normal milk to have the canbequilty variable set to 1.
So normal milk can still spawn as quilty but clownspider milk cannot. :)

Reply:
i know, in the line i have selected you are checking if the variable is less then 1 and not if its 1

Reply:
So I should change it to if(canbequilty == 1) ???????

Reply:
yes

Reply:
Thanks for your help moon! :)

Did some new changes in the latest commit; tested it out and normal milk can spawn in as quilty while my custom milk still spawns in correctly. Take care buddy.